docs/docs/faq/question_05.md

```diff
@@ -0,0 +1,46 @@
+
+# How should I save and load objects in `sbi`?
+
+`NeuralPosterior` objects are picklable.
+```python
+import pickle
+
+# ... run inference
+posterior = inference.build_posterior()
+
+with open("my_path.pkl", "wb"):
+    pickle.dump(posterior)
+```
+
+`NeuralInference` objects are not picklable. There are two workarounds:
+
+- Pickle with `dill` (has to be installed with `pip install dill` first)
+```python
+import dill as pickle
+
+inference = SNPE(prior)
+# ... run inference
+
+with open("my_path.pkl", "wb"):
+    pickle.dump(inference)
+```
+
+- Delete un-picklable attributes and serialize with pickle. Using this option, you will not be able to use the `retrain_from_scratch` feature and you can only use the default `SummaryWriter`.
+```python
+import pickle
+
+inference = SNPE(prior)
```

Comment:
same comment as above, I suggest to include code that is known from the tutorials.